portfolio.html

```diff
@@ -0,0 +1,70 @@
+<!Doctype html>
+<html lang="en" ontouchmove>
+	<head>
+		<title>Portfolio ALex F.</title>
+		<meta charset="UTF-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<link href="./css/normalise.css" type="text/css" rel="stylesheet">
+		<link href="./css/style.css" type="text/css" rel="stylesheet">
+		
+	</head>
+	
+	<header>
+		<nav class="container navbar"> <!--Navbar start-->
+			<h1>Alex Ferreira</h1>
+			<div class="link">
+				<a href="./index.html" class="navlink">Home</a>
+				<a href="./portfolio.html"class="navlink current">Portfolio</a>
+				<a href="./aboutme.html"class="navlink">About me</a>
+				<a href="./contact.html"class="navlink">Contact</a>
+			</div>
+			
+		</nav> <!--Navbar end-->
+	</header>
+	
+	<body class="body-image"> <!--body start-->
+		
+		<main id="portfolio-body-image"> <!--main start-->
+			<div class="content-box">
+				<h2>- Portfolio -</h2>
+			</div>
+			
+			<div class="planet-projet" id="projet-left" id="p1">
+				<h3>Project Nam</h3>
+				<img src="./images-logo/Baren.png" alt="Projet 1">
+				<p class="project-p"> "Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit..."
+"There is no one who loves pain itself, who seeks after it and wants to have it, simply because it is pain...""Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit..."</p>
+			</div>
+			
+			<div class="planet-projet" id="projet-right" id="p2">
+				<h3>Project Nam</h3>
+				<img src="./images-logo/Lava.png" alt="Projet 1">
+				<p class="project-p"> "Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit..."
+"There is no one who loves pain itself, who seeks after it and wants to have it, simply because it is pain..."</p>
+			</div>
+			
+			<div class="planet-projet" id="projet-left" id="p3">
```

Comment:
why are you using an id? you cant use the same ID more than once on a page, so technically this is invalid HTML. please use classes